docs/root/configuration/http/http_filters/router_filter.rst

```diff
@@ -240,7 +240,8 @@ x-envoy-upstream-rq-per-try-timeout-ms
 
 Setting this header on egress requests will cause Envoy to set a *per try* timeout on routed
 requests. This timeout must be <= the global route timeout (see
-:ref:`config_http_filters_router_x-envoy-upstream-rq-timeout-ms`) or it is ignored. This allows a
```

Comment:
Actually, could you reword this to something like `If a global timeout is defined .. this must be <= the global timeout or it will be ignored`? I think it would read a bit cleaner

Reply:
Agreed, that will flow better. Also, I just noticed that `<=` on the next line is incorrect. It should be `<`. If they're equal, it will still be ignored. I'll fix that while I'm touching it.